From aebfaec53af9760aacf4cdc20cbaa8126fac6892 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cristian=20One=C8=9B?= <onet.cristian@gmail.com>
Date: Wed, 21 Nov 2012 08:59:57 +0200
Subject: [PATCH 05/22] Use the 'Higlight' background color for warnings since
 the 'Infobackground' can be really close to the text color. If the same would
 happen to the 'Higlight' background color we'll notice it a lot faster since
 that is also used in the summary headers.

REVIEW: 107397
(cherry picked from commit ad1d684caf8a95f1be289647012b61b5833b04b8)
---
 kmymoney/html/kmymoney.css | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kmymoney/html/kmymoney.css b/kmymoney/html/kmymoney.css
index e44ae0a..ab63d53 100644
--- a/kmymoney/html/kmymoney.css
+++ b/kmymoney/html/kmymoney.css
@@ -103,11 +103,11 @@ body#summaryview { /* set body attributes that affect only the summary view (and
 
   .warningtitle { /*<tr> -set warning background color -overdue payments -items of this class also belong to the class called "itemtitle"*/
     color:red;/*set programatically, not by css*/
-    background-color: infobackground;
+    background-color: HighLight;
   }
 
   .warning { /*<tr> -set warning background color -items of this class also belong to the class called "item"*/
-    background-color: infobackground;
+    background-color: HighLight;
   }
 
   th.left, td.left {/*rule to align table header and table data cells*/
-- 
1.8.1.4

